integreat_cms/cms/models/events/event_translation.py

```diff
@@ -83,6 +85,24 @@ def backend_edit_link(self) -> str:
             },
         )
 
+    @cached_property
+    def link_title(self) -> Element | str:
```

Comment:
And I think we could do some refactoring here to reduce code duplication.
The implementation is almost the same for all content types. I think we could basically keep one default method in abstract_content_translation.py and only re-assign the default icon (null for pages, not-null for events and locations)